_data/categories.yml

```diff
@@ -57,3 +57,7 @@
 - name: Testing
   description: Quickly execute API request and validate responses on the fly through command line or GUI interfaces.
   slug: testing
+
+- name: Data Modeling
```

Comment:
I hear you, supermodel ia a tricky category as it does many things. OpenAPI is about describing the service model and data model, so this category is very wide. The wording could describe editors, data validators, a DSL, or many other things... I'm not sure what to do.

Reply:
Without getting too deep .... I am ok to settle on DSL, if you prefer. However I am not really sure what is the difference (by description) between DSL and text editors...

Taxonomy is hard.

Maybe we can call it all just "language"? Or "authoring" as in authoring API design – OAS document?

Reply:
taxonomy _is_ hard.

Going by my definitions here, but an "authoring" category could encompass all text/gui editors, annotations, and DSL.

Text editors are just editors which help you work on OpenAPI. A DSL is another language that you write instead of OpenAPI, which can pop out OpenAPI for you. I think that might be the closest to what supermodel is up to, at least in this context (as it does other stuff like make UIs etc).

Reply:
@philsturgeon Ok, I've changed it to `dsl` and removed the new category. Hope this works. Cheers!